pom.xml

```diff
@@ -83,13 +88,30 @@
       <version>5.0.11</version>
       <scope>test</scope>
     </dependency>
+    <!-- fix transitive dependency conflict -->
+    <dependency>
+      <groupId>org.jenkins-ci.main</groupId>
+      <artifactId>jenkins-test-harness</artifactId>
+      <version>2034.v41c9cb_349299</version>
+      <scope>test</scope>
+      <exclusions>
```

Comment:
Where does the included test-harness come from? And why is it a conflict?

Reply:
Updating `org.jenkins-ci.plugins.plugin` to the latest version (4.71) fixed this issue.